tensorflow_serving/servables/tensorflow/util.cc

```diff
@@ -47,6 +47,48 @@ auto* example_count_total = monitoring::Counter<1>::New(
     "/tensorflow/serving/request_example_count_total",
     "The total number of tensorflow.Examples.", "model");
 
+// Metrics by model
+auto* model_request_count_total = monitoring::Counter<1>::New(
```

Comment:
change this to Counter<2> to collect per-model+status count. so you can collect success and errors together.